pkg/registry/apis/provisioning/jobs/worker.go

```diff
@@ -169,6 +169,6 @@ func (g *JobWorker) Process(ctx context.Context, job provisioning.Job) (*provisi
 	}
 
 	return &provisioning.JobStatus{
-		State: "finished", // success
```

Comment:
I don't know if we have a specific reason for this saying `"finished"` rather than `"success"`, so I changed it over. I can change the finished constant to be `"finished"` in its value instead.

Reply:
weird -- we had:
```
// Finished with success
JobStateFinished JobState = "success"
```
I changed that in: https://github.com/grafana/grafana/pull/98669